_docs/setup/kubernetes/ansible-install.md

```diff
@@ -119,4 +119,4 @@ It should be noted that when Jaeger is enabled, Zipkin is disabled whether or no
 ## Adding istioctl to PATH
 
 After executing the playbook if it is desired that the `istioctl` command line tool be added to the PATH,
-search for `Add Istio to PATH` in the output and execute the commands that are outputted  
\ No newline at end of file
+search for `Add Istio to PATH` in the output and execute the commands that are outputted  
```

Comment:
I can't see / tell what changed here between the 2 lines ?
(also is "outputted" valid?)

Reply:
Changed "Instructions on" to "Instructions for" in this file.  I have no idea why git changed line 122 in the diff.  I'm not sure how to revert the line 122 change.  Any ideas?

Reply:
re outputted - the Ansible doc could use a tech writer editing.  I can take that on as a separate PR if you like.  Goal of this PR is only to unify the filenames, titles, overview descriptors on the main page rendering.